vnext/template/shared-lib/proj/ExperimentalFeatures.props

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup Label="Microsoft.ReactNative Experimental Features">
+    
+    <!--
+      Changes compilation to assume use of WinUI 3 instead of System XAML.
+      Requires creation of new project.
+
+      See https://microsoft.github.io/react-native-windows/docs/0.64/winui3
+    -->
+    <UseWinUI3>{{useWinUI3}}</UseWinUI3>
```

Comment:
@asklar: To continue the discussion of your PR....
How will this work for community modules that need to support both winui2 and winui3...
I thought this prop file would be part of the app and then included in all libraries that are build as part of the solution (similar to build.directory.props) That way the app can toggle between modes (and then future nuget as well) and all the modules that support both will 'just work' as they can adapt as needed...
Having this file next to the lib makes it hide / toggle.. Perhaps this .props should try to include one in the parent directory (recursively) and set WinUI3 if not already set in a parent?

Reply:
Updated in my PR after talking to danny - the libraries' buildflags now will just redirect to the app's